pallets/ajuna-awesome-avatars/src/lib.rs

```diff
@@ -165,8 +165,18 @@ pub mod pallet {
 	pub type CollectionId<T: Config> = StorageValue<_, CollectionIdOf<T>, OptionQuery>;
 
 	#[pallet::storage]
-	pub type Accounts<T: Config> =
-		StorageMap<_, Identity, T::AccountId, AccountInfo<T::BlockNumber>, ValueQuery>;
+	pub type Players<T: Config> = StorageMap<_, Identity, T::AccountId, PlayerConfig, ValueQuery>;
```

Comment:
Can we choose a more descriptive name than `Players`? Because I can't tell what this storage is supposed to contain.

Reply:
Chnaged name to `PlayerConfigs`!